docs/self-managed/platform-deployment/docker.md

```diff
@@ -149,6 +154,13 @@ Some configuration properties are optional and have default values. See a descri
 | SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI   | Authentication for the Public REST API using a resource server to validate the JWT token. Complete URI to get public keys for JWT validation.                                              | null          |
 | OPTIMIZE_API_ACCESS_TOKEN                               | Authentication for the Public REST API using a static shared token. Will be ignored if SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI is also set.                                  | null          |
 
+\* Only relevant when `CAMUNDA_OPTIMIZE_DATABASE` is either undefined or has the value `elasticsearch`.<br/>
+\*\* Only relevant when `CAMUNDA_OPTIMIZE_DATABASE` has the value `opensearch`.
+
+:::note
+Setting `CAMUNDA_OPTIMIZE_DATABASE` to `opensearch` is currently only possible for `ccsm` mode. Moreover, OpenSearch support in Optimize is limited to data import and the raw data report. The remaining functionality will be delivered with upcoming patches.
```

Comment:
`currently only possible for `ccsm` mode`. Why do we need to say this? This whole page is only targeting C8SM users, who will already be running it with this profile?

Reply:
True, removing it